SECURITY.md

```diff
@@ -74,7 +73,8 @@ MEDIUM [$10,000 USD]
 
 LOW [$5,000 USD]
 
--   Sending a transaction that triggers invariant violation error code in unmodified validator software
+1. Sending a transaction that triggers invariant violation error code in unmodified validator software
+2. A remote call that crashes a Sui fullnode
```

Comment:
FYI - you can just use 1. for all entries in an ordered list and it will render correctly.